src/test/ui/const_prop/inline_spans.stderr

```diff
@@ -0,0 +1,13 @@
+error: this arithmetic operation will overflow
+  --> $DIR/inline_spans.rs:7:13
+   |
+LL |     let _ = add(u8::MAX, 1);
+   |             ^^^^^^^^^^^^^^^ attempt to compute `u8::MAX + 1_u8` which would overflow
+...
+LL |     x + y
+   |     ----- in the inlined copy of this
```

Comment:
Maybe say?

```suggestion
   |     ----- in the inlined copy of this expression
```

Reply:
It's not necessarily an expression. It could be an arbitrary statement. A generic version could be `in the inlined copy of this code`

Reply:
I went with `in the inlined copy of this code`